html-css/.github/workflows/linters_css.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Lighthouse
+        run: npm install -g @lhci/cli@0.4.x
+      - name: Lighthouse Report
+        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
+  webhint:
+    name: Webhint
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Webhint
+        run: |
+          npm install --save-dev hint@6.0.x
+          [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.hintrc
+      - name: Webhint Report
+        run: npx hint --telemetry=off .
+  stylelint:
+    name: Stylelint
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.3.x stylelint-config-standard@20.0.x
```

Comment:
If we skip the installation of `stylelint-scss`, which makes sense as it will not be needed for CSS, then we will need also to modify ths Stylelint config provided and have two of those, one for CSS and one for SCSS, as the current one uses that plugin:

https://github.com/microverseinc/linters-config/blob/e1c8370b36c7ddc60123479e8b5a3b4b897ba6f5/html-css/.stylelintrc.json#L3

Reply:
Maybe we could just give indications on how to modify the configuration files to lint SCSS files, in this case, removing the plugin 🤔